docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-feature-integration.md

```diff
@@ -1181,6 +1188,7 @@ class MerchantProductOfferStorageDependencyProvider extends SprykerMerchantProdu
     {
         return [
             new ProductOfferReferenceStrategyPlugin(),
+            new MerchantProductProductOfferReferenceStrategyPlugin(),
```

Comment:
As far as this one plugin is registered in `MerchantProductStorage` module, which is a part of `MerchantProduct` feature. I propose to move it there and provide a link in this IG for the case when customer wants to install `Product Offer` feature after `Merchant Product` feature

Reply:
must be placed in Marketplace Product + Marketplace Product Offer